etcdserver/api/membership/member.go

```diff
@@ -35,6 +35,8 @@ type RaftAttributes struct {
 	// PeerURLs is the list of peers in the raft cluster.
 	// TODO(philips): ensure these are URLs
 	PeerURLs []string `json:"peerURLs"`
+	// IsLearner indicates if the member is raft learner.
+	IsLearner bool `json:"isLearner"`
```

Comment:
Do we need to allow this to be absent for backward compatibility?

Reply:
Good catch. I think it is safer with "omitempty".